src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,10 @@
+const ArticleCard = ({ article }) => {
+  return (
+    <section className="article-card">
+      <h3 className="article-title">{article.title}</h3>
+      <article className="article-body">{article.body}</article>
```

Comment:
This specific end point we requested that the body was NOT returned so that listing all articles didn't look bloated. This should not be here.

Reply:
fixed